### ISSUE TITLE
interopNamespace added when interop is disabled

### ISSUE THREAD

**Issue by gingur** [rollup/rollup#3288]
- Rollup Version: 1.27.9
- Operating System (or Browser): Windows 10
- Node Version: 10.15.3

### How Do We Reproduce?

```js
// rollup.config.js
config.output.format = 'cjs';
config.output.interop = false;

// entry
import('someExternal');
```
### Expected Behavior
```js
new Promise(function (resolve) { resolve(require('someExternal')); });
```

### Actual Behavior
```js
new Promise(function (resolve) { resolve(_interopNamespace(require('someExternal'))); });
```

**Comment by shellscape** [comment]
Thanks for opening an issue, and apologies that it took a while to get a response. Citing the issue template:

>   Issues without minimal reproductions will be closed! Please provide one by:
>   1. Using the REPL at https://rollupjs.org/repl/, or
>   2. Work to isolate the problem and provide the exact steps in this issue, or
>   3. Provide a minimal repository link (Read https://git.io/fNzHA for instructions).
>      These may take more time to triage than the other options.

Please add a reproduction and we'll be happy to triage further.

**Comment by gingur** [comment]
@shellscape pretty sure I provided #2, I isolated the problem to a config flag, and provided the line of code. If you require #1 or #3 only, I would suggest removing #2 from your template.

**Comment by gingur** [comment]
Pretty sure the REPL doesn't use a file named `rollup.config.js`, but this is essentially the source https://tinyurl.com/wecehq3

**Comment by lukastaegert** [comment]
Yes, I think the `interop` flag was not considered when creating the interop logic for dynamic namespaces. Marking this as a bug as the suggested behaviour makes a lot of sense. I do not think it would be a big change so PR definitely welcome.

**Comment by gingur** [comment]
update:, still an issue in 2.2.0

**Comment by lukastaegert** [comment]
Fix at #3710
